src/btree/bt_delete.c

```diff
@@ -363,7 +363,7 @@ __wt_delete_page_instantiate(WT_SESSION_IMPL *session, WT_REF *ref)
 	 * able to do so.)
```

Comment:
I think this comment should be updated - the code will now always mark the tree dirty unless it's read only.